lib/frame-manager.js

```diff
@@ -42,10 +41,15 @@ function FrameManager(window) {
     if (!self.listenerCount('data')) {
       parent.emit('log', 'unsubscribing from browser window frames')
       window.webContents.endFrameSubscription();
-      subscribed = false;
     }
   }
 
+  function cancelFrame() {
+    requestedFrame = false;
+    clearTimeout(frameRequestTimeout);
+    self.emit('data', null);
+  }
+
```

Comment:
Ah, good insight. I didn't even see that o_o

If we are going for conciseness, we could probably do 1 better and remove `cancelFrame()` altogether (i.e. call `receiveFrame(null);` at those locations).

If we are going for clarity (my preference), I sometimes will sacrifice DRYness for making the intent clear (i.e. we mean to cancel the frame request and emit `null` data). In the concise scenario and another contributor reading the code, they might be uncertain if we actually wanted to emit `null` or not =/